ocsp_status_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"crypto"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+)
+
+const (
+	GoodResp = "./testdata/resps/oldfbresp"
+	GoodCert = "./testdata/certs/google.der"
+	ExpiredCert = "./testdata/certs/expiredcert.der"
+	NoIssuingURLCert = "./testdata/certs/rootcert.der"
+	GoodURL = "google.com:443"
+	RevokedURL = "revoked.grc.com:443"
+	BadURL = "blah.blah.blah"
+)
+
+// TestMain sets up the testing framework
+func TestMain(m *testing.M) {
+	// discard logging
+	log.SetOutput(ioutil.Discard)
+	os.Exit(m.Run())
+}
+
+// TestCheckFromFile tests checkFromFile
+func TestCheckFromFile(t *testing.T) {
+	err := checkFromFile(GoodResp)
+	if err != nil {
+		t.Errorf("Got error reading good response: %s", err.Error())
+	}
+
+	err = checkFromFile(GoodCert)
+	if err == nil {
+		t.Errorf("Should have gotten error reading file that is not an OCSP response")
+	}
```

Comment:
Can you break up this into two different tests so that if `checkFromFile(GoodResp)` fails, it will still run the test for `checkFromFile(GoodCert)`? I don't know what the test output looks like, but it will also probably allow you to see the results more granularly. Also I have the same comment for all of the test cases below. Can they be broken up so that you only have to check `if err != nil {...}` once per test case.

Reply:
The way I currently have it set up, I think it will run all tests even if a previous test fails; `t.Errorf` is just how you indicate a test fails with the Go testing package.

Reply:
after discussions I think this will happen in a follow up PR.